src/main/webapp/index.html

```diff
@@ -50,51 +49,7 @@
                     </ul>
                 </div>
             </div>
-            <div class="product">
-                <div class="card w-75 itemBox item1">
-                    <div class="card-header">
-                        Intro to Python
-                    </div>
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
-                        <a href="#" class="btn btn-primary">Button</a>
-                    </div>
-                </div>
-
-                <div class="card w-75 itemBox item2">
-                    <div class="card-header">
-                        The JavaScript Bundle
-                    </div>
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
-                        <a href="#" class="btn btn-primary">Button</a>
-                    </div>
-                </div>
-
-                <div class="card w-75 itemBox item3">
-                    <div class="card-header">
-                        C++
-                    </div>
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
-                        <a href="#" class="btn btn-primary">Button</a>
-                    </div>
-                </div>
-
-                <div class="card w-75 itemBox item4">
-                    <div class="card-header">
-                        Intro to Python
-                    </div>
-                    <div class="card-body">
-                        <h5 class="card-title">Card title</h5>
-                        <p class="card-text">With supporting text below as a natural lead-in to additional content.</p>
-                        <a href="#" class="btn btn-primary">Button</a>
-                    </div>
-                </div>
-            </div>
+            <div id="product"></div>
```

Comment:
You could have an id and a class. We usually use classes for styling.

Reply:
That makes sense, I'll do that. I changed it to an _id_ because I was using _getElementByID_ in the script to return this element.